torch/csrc/jit/passes/utils/op_registry.cpp

```diff
@@ -0,0 +1,72 @@
+      "aten::_convolution(Tensor input, Tensor weight, Tensor? bias, int[] stride, int[] padding, int[] dilation, bool transposed, int[] output_padding, int groups, bool benchmark, bool deterministic, bool cudnn_enabled, bool allow_tf32) -> Tensor",
+      "aten::adaptive_avg_pool1d(Tensor self, int[] output_size) -> Tensor",
+      "aten::adaptive_avg_pool2d(Tensor self, int[] output_size) -> Tensor",
+      "aten::adaptive_avg_pool3d(Tensor self, int[] output_size) -> Tensor",
+      "aten::avg_pool1d(Tensor self, int[] kernel_size, int[] stride, int[] padding, bool ceil_mode, bool count_include_pad) -> Tensor",
+      "aten::avg_pool2d(Tensor self, int[] kernel_size, int[] stride, int[] padding, bool ceil_mode, bool count_include_pad, int? divisor_override) -> Tensor",
+      "aten::avg_pool3d(Tensor self, int[] kernel_size, int[] stride, int[] padding, bool ceil_mode, bool count_include_pad, int? divisor_override) -> Tensor",
+      "aten::max_pool1d(Tensor self, int[] kernel_size, int[] stride, int[] padding, int[] dilation, bool ceil_mode) -> Tensor",
+      "aten::max_pool2d(Tensor self, int[] kernel_size, int[] stride, int[] padding, int[] dilation, bool ceil_mode) -> Tensor",
+      "aten::max_pool3d(Tensor self, int[] kernel_size, int[] stride, int[] padding, int[] dilation, bool ceil_mode) -> Tensor",
+      "aten::max_unpool2d(Tensor self, Tensor indices, int[] output_size) -> Tensor",
+      "aten::max_unpool3d(Tensor self, Tensor indices, int[] output_size, int[] stride, int[] padding) -> Tensor",
+      "aten::reflection_pad1d(Tensor self, int[] padding) -> Tensor",
+      "aten::reflection_pad2d(Tensor self, int[] padding) -> Tensor",
+      "aten::reflection_pad3d(Tensor self, int[] padding) -> Tensor",
+      "aten::replication_pad1d(Tensor self, int[] padding) -> Tensor",
+      "aten::replication_pad2d(Tensor self, int[] padding) -> Tensor",
+      "aten::replication_pad3d(Tensor self, int[] padding) -> Tensor",
+      "aten::upsample_bilinear2d(Tensor self, int[] output_size, bool align_corners, float? scales_h, float? scales_w) -> Tensor",
+      "aten::upsample_linear1d(Tensor self, int[] output_size, bool align_corners, float? scales) -> Tensor",
+      "aten::upsample_nearest1d(Tensor self, int[] output_size, float? scales) -> Tensor",
+      "aten::upsample_nearest2d(Tensor self, int[] output_size, float? scales_h, float? scales_w) -> Tensor",
+      "aten::upsample_nearest3d(Tensor self, int[] output_size, float? scales_d, float? scales_h, float? scales_w) -> Tensor",
+      "aten::upsample_trilinear3d(Tensor self, int[] output_size, bool align_corners, float? scales_d, float? scales_h, float? scales_w) -> Tensor",
+      "aten::prelu(Tensor self, Tensor weight) -> Tensor",
+  });
+  return ops;
```

Comment:
I wonder why they didn't use a plain reference instead, but it isn't something you changed or introduced.